code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -658,11 +658,11 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
-/datum/design/borg_upgrade_ashplating
-	name = "Cyborg Upgrade (Ash Storm Plating)"
-	id = "borg_upgrade_ashplating"
+/datum/design/borg_upgrade_lavaproof
+	name = "Cyborg Upgrade (Lavaproof Tracks)"
+	id = "borg_upgrade_lavaproof"
 	build_type = MECHFAB
-	build_path = /obj/item/borg/upgrade/ashplating
+	build_path = /obj/item/borg/upgrade/lavaproof
 	req_tech = list("plasmatech" = 4, "materials" = 4, "engineering" = 4)
 	materials = list(MAT_METAL = 8000, MAT_PLASMA = 10000)
```

Comment:
Atomizing PRs is annoying even without tokens. Instead of having to get 1 large PR merged, you had to get 20 small PRs merged.